src/framework/preferences/js/ArrowScrolling.js

```diff
@@ -30,7 +30,7 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         onScrollDelay: 100, // in ms, used to set the delay for debouncing the scroll event relay
         model: {
             // panelMaxIndex: null, // determined by the number of panels calculated after the onPrefsEditorMarkupReady event fired
-            panelIndex: 0
+            // panelIndex: 0 // the index of the panel to open on
```

Comment:
We seem to have picked up some stray material from another pull - can we rebase this against current master?

Reply:
Actually I needed to comment that line out in order for the panelIndex to be properly pulled in from the store. Otherwise this was overriding things back to 0, because of the model relay I believe.

Reply:
But there's no connection between the work in this pull and the ArrowScrolling work, right?

Reply:
Oh sorry, I see that there is. We should try to put the default value back in, and prevent it from propagating out with an excludeSource: "init" directive on the relay

Reply:
@amb26 I think I'm remembering more of the problem. If there is a default value, it will be applied instead of the value from the cookie (or other datastore). If exclude init, then you can't pass in a  panelIndex through the component settings to have it default to a different panel.

Although now that I'm testing this out by uncommenting the default value for panelIndex, I'm getting a strange error at https://github.com/jobara/infusion/blob/ca255c90147329a2a277c3c47efc267ac6f913f3/src/framework/preferences/js/PrefsEditor.js#L352 It seems that the changeMap has the value "ADD". This results in the following error "DataBinding.js:496 Uncaught TypeError: Cannot read property '0' of undefined"

Reply:
That's actually an ok value for a ChangeMap and the prefsEditor code has a bug. A ChangeMap isn't a flat map, it's an object isomorphic to the model with values "ADD" and "DELETE" at the locations where changes have occurred

Reply:
@amb26 I've tried to exclude init for the panelIndex relay, but it seems to cause the issue I mentioned above, where it ignores values for the panelIndex in the components defaults.

Reply:
OK, this is a pretty interesting issue - do you think you could write up the details of how this situation works as a JIRA, with a title something like "model relay system cannot distinguish between different sources of initial values"

Reply:
@amb26 I've filed a JIRA https://issues.fluidproject.org/browse/FLUID-6249 Let me know if you'd like any more information provided.